readthedocs/proxito/views/serve.py

```diff
@@ -62,7 +62,8 @@ def get(self,
             # This is since the final URL will check for authz.
             self.cache_request = True
 
-        is_external = getattr(request, "external_domain", False)
+        unresolved_domain = request.unresolved_domain
+        is_external = unresolved_domain and unresolved_domain.is_from_external_domain
```

Comment:
I think we could assume that the unresolved domain will always be set in some views, changing this...